internal/webserver/server.go

```diff
@@ -152,8 +152,16 @@ func (webserver *WebServer) listenAndServe(serviceTimeout time.Duration, errChan
 		ozServiceName := zerotrust.OpenZitiServicePrefix + webserver.serviceName
 		lc.Infof("Using OpenZiti service name: %s", ozServiceName)
 		lc.Infof("listening on overlay network. ListenMode '%s' at %s", listenMode, addr)
+		for startupTimer.HasNotElapsed() {
+			endpoint, err = cb.registry.GetServiceEndpoint(serviceKey)
+			if err == nil {
+				break
+			}
+
+			lc.Warnf("unable to Get service endpoint for '%s': %s. retrying...", serviceKey, err.Error())
+			startupTimer.SleepForInterval()
+		}
```

Comment:
where do those variables come from?

```
[2024-07-10T20:43:43.795Z] # github.com/edgexfoundry/app-functions-sdk-go/v3/internal/webserver
[2024-07-10T20:43:43.795Z] ../internal/webserver/server.go:155:7: undefined: startupTimer
[2024-07-10T20:43:43.795Z] ../internal/webserver/server.go:156:4: undefined: endpoint
[2024-07-10T20:43:43.795Z] ../internal/webserver/server.go:156:20: undefined: cb
[2024-07-10T20:43:43.795Z] ../internal/webserver/server.go:156:51: undefined: serviceKey
[2024-07-10T20:43:43.795Z] ../internal/webserver/server.go:161:73: undefined: serviceKey
[2024-07-10T20:43:43.795Z] ../internal/webserver/server.go:162:4: undefined: startupTimer
[2024-07-10T20:43:43.795Z] make[1]: *** [Makefile:42: build] Error 1
```

Reply:
wow. I'm sorry, I totally missed that this didn't compile. I don't know how I put this pr up. I'll take this down since it's probably not needed at this time.